src/templates/components/container.html

```diff
@@ -0,0 +1,12 @@
+<div class="panel panel-default" ng-controller="formioContainerComponent">
```

Comment:
This should not be a panel.  It should be an invisible container of fields so it should only have a ```<div>``` wrapper.